### DISCUSSION TITLE
Adding a background image from Unsplash?

### DISCUSSION THREAD

**Discussion by fluffeuk** [monkeytypegame/monkeytype#5260] [Questions] [answered]
I tried to add a background, i used this link: https://images.unsplash.com/photo-1709603945846-6901ed447ecd?q=80&w=3029&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D 

For some reason monkeytype says that the link is invalid? Must I format the link a certain way? Or is unsplash just not supported?

**Comment by fehmer** [comment, marked as answer]
try  https://images.unsplash.com/photo-1709603945846-6901ed447ecd?q=80&w=3029&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D.png

**Reply by fluffeuk** [reply]
Perfect. How'd you do it?

**Reply by fehmer** [reply]
just add `.png` at the end
